bin/http_server.re

```diff
@@ -15,6 +15,9 @@ open Protocol;
 open Node;
 open Networking;
 
+Logs.set_reporter(Logs_fmt.reporter());
+Logs.set_level(Some(Logs.Debug));
```

Comment:
I'd prefer if this is Error by default, perhaps Warning. By making it Debug, we treat all levels of the logs with the same priority

Reply:
Spoke to early: Line 19 shouldn't be necessary - Logs_cli should set the default level to warning already.

Reply:
Ignore me. http_server isn't using Logs_cli